indigo_content_api/v3/urls.py

```diff
@@ -0,0 +1,38 @@
+from django.urls import include, path, re_path
+from rest_framework.routers import DefaultRouter
+
+from indigo_content_api.v2 import views as views_v2
+from . import views
+
+router = DefaultRouter(trailing_slash=False)
+router.register(r'countries', views_v2.CountryViewSet, basename='country')
+router.register(r'taxonomy_topics', views_v2.TaxonomyTopicView, basename='taxonomy_topic')
+
+
+# TODO: inherit v2 urlpatterns, just override PublishedDocumentDetailView's path?
+urlpatterns = [
```

Comment:
another option is to pull in the URLs from v2, and put the new and overriding urls at the start of this list, something like this:

```
urlpatterns = [
  '.../commencements.json',
  '/akn/...', views_v3.OverriddenPublishedDocumentViewWithNewSerializer,
] + v2_urlpatterns
```

that'll save you from duplicating everything.

Reply:
thanks!